src/main/java/mate/academy/spring/service/RoleService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.model.Role;
+
+public interface RoleService {
+    Role add(Role adminRole);
+
+    Role getByName(Role.RoleName name);
```

Comment:
Here we also should pass String:
```suggestion
    Role getByName(String name);
```